content/en/developers/custom_checks/write_agent_check.md

```diff
@@ -33,29 +33,34 @@ instances:
   [{}]
 {{< /code-block >}}
 4. Create a check file in the `checks.d` directory. Name the file `custom_checkvalue.py`.
-   <div class="alert alert-info">The names of the configuration and check files must match. If your check is called `custom_checkvalue.py`, your configuration file *must* be named `custom_checkvalue.yaml`.</div> 
+   
+   <div class="alert alert-info">
+     <strong>Naming your checks:</strong>
+     <ul>
+       <li>It's a good idea to prefix your check with <code>custom_</code> to avoid conflicts with the name of a preexisting Datadog Agent integration. For example, if you have a custom Postfix check, name your check files <code>custom_postfix.py</code> and <code>custom_postfix.yaml</code> instead of <code>postfix.py</code> and <code>postfix.yaml</code>.</li>
```

Comment:
```suggestion
       <li>It's a good idea to prefix your check with <code>custom_</code> to avoid conflicts with the name of a pre-existing Datadog Agent integration. For example, if you have a custom Postfix check, name your check files <code>custom_postfix.py</code> and <code>custom_postfix.yaml</code> instead of <code>postfix.py</code> and <code>postfix.yaml</code>.</li>
```

Un-sure if this should by hyphenated ? I think either form is accepted so up to you.

Reply:
Good call. It looks like we use it hyphenated in other docs, so I'll do that for consistency 👍